courtroom.cpp

```diff
@@ -980,15 +980,36 @@ void Courtroom::handle_chatmessage_3()
   QString f_emote = m_chatmessage[EMOTE];
 
   ui_vp_showname_image->show();
+
+  // Check for any of 9 possible ways that showname images are
   QVector<QString> exts = {".png", ".jpg", ".bmp"};
+  // 3 places (in order)
+  // 1. Variant folder in character folder in theme folder
+  // 2. Character folder in folder folder
+  // 3. Character folder
+  QStringList directories = {
+      ao_app->get_theme_variant_path("characters/" + f_char.toLower() +
```

Comment:
```suggestion
      ao_app->get_theme_variant_path("characters/" + f_char +
```

Though this may call for a reformatting, so I suggest not accepting it here on Github.

Reply:
This was just me forgetting to remove the .toLower() from the old PR. What do you mean with the reformatting tho?

Reply:
Github won't `clang-format` if you just press the "Commit suggestion" button, and I got a feeling that removing that `toLower()` would need that line reformatted.
Though I'm not sure.